0001-kaa-protocol/README.md

```diff
@@ -159,6 +162,10 @@ In that case, the endpoint is manually added to the server, the server allocates
 It's good to keep topic names short.
 Maybe we can introduce topic aliases, so user won't need to use full-length topic name.
 
+### Status topics
+Would it make sense to move `/status` suffix to some place in the middle of the MQTT path (like between the common and extension-specific parts) instead of the end?
```

Comment:
I'll copy this comment into the RFC AI so that we don't lose the context, but will not change the spec.